README.md

```diff
@@ -8,10 +8,19 @@ For deploying this tool to a production server, see [commcare-sync-ansible](http
 
 ## Developer Setup
 
+### Prerequisites
+- Python 3.8
+    On Ubuntu:
+    ```bash
+    sudo apt-get install python3.8 python3.8-dev
+    ```
+- Postgres
+- Redis
```

Comment:
would we think of adding these as docker packages here?

Reply:
What do you mean, exactly? In my local env I just use the same docker images as HQ.

Reply:
so did I, but I feel like this is a completely standalone product, so we should probably have instructions on how to run this without HQ locally.

Reply:
That makes sense. I would have thought the simplest thing to do is just install postgres natively, though I have no objections to adding docker instructions as well.